test/test_component.rb

```diff
@@ -113,8 +113,8 @@
         before do
             @task_m = Syskit::TaskContext.new_submodel do
                 output_port "out", "int"
-                dynamic_output_port /\w+_out/, "bool"
-                dynamic_input_port /\w+_in/, "double"
+                dynamic_output_port(/\w+_out/, "bool")
+                dynamic_input_port(/\w+_in/, "double")
```

Comment:
?

Reply:
Ah, didnt notice that auto format changed this part as well